src/main/java/types/Cluster.java

```diff
@@ -522,4 +522,32 @@
      * @since 4.4.5
      */
     FipsMode fipsMode();
+
+    /**
+     * Indicates if an upgrade has been started for the cluster.
+     *
+     * @author Scott Dickerson <sdickers@redhat.com>
+     * @date 17 Mar 2022
+     * @since 4.5.0
+     */
+    Boolean upgradeInProgress();
+
+    /**
+     * If an upgrade is in progress, the upgrade's reported percent complete.
+     *
+     * @author Scott Dickerson <sdickers@redhat.com>
+     * @date 17 Mar 2022
+     * @since 4.5.0
+     */
+    Integer upgradeProgress();
```

Comment:
Based on Ori's comment, maybe worth renaming this to upgradeProgressPercent

Reply:
I could go either way on the naming.

Reply:
It's always ideal if the name is so clear that the user doesn't have to refer to documentation. 

   <upgrade_progress>15</upgrade_progress>

is ambiguous (15 what - minutes? minutes-left? percent?)

While something like:

<upgrade_percent_completed>15</upgrade_percent_completed>

is immediately clear upon reading and the user doesn't have to bother with documentation.

So I think there is value in changing this